shell/components/RelatedWorkloadsTable.vue

```diff
@@ -1,50 +0,0 @@
-<script>

```

Comment:
this was added in https://github.com/rancher/dashboard/pull/2342, but the usage was removed in https://github.com/rancher/dashboard/pull/2436. Given it does a findAll on all workload types this seems like a good pr to remove it